src/Interface/Modules/Render/ES/SRInterface.cc

```diff
@@ -346,7 +348,7 @@ namespace SCIRun {
 
     //----------------------------------------------------------------------------------------------
     void SRInterface::select(const glm::ivec2& pos,
-      std::list<Graphics::Datatypes::GeometryHandle> &objList,
+      std::vector<Graphics::Datatypes::WidgetHandle> &objList,
```

Comment:
As soon as a data structure changes, it should be extracted into a typedef/type alias for readability/robustness